scripts/run-azurite.ps1

```diff
@@ -0,0 +1,100 @@
+azurite.
+
+#>
+
+<#
+.SYNOPSIS
+This is a Powershell script to run previously installed azurite on Windows.
+
+.DESCRIPTION
+This is a Powershell script to run previously installed azurite server on Windows.
+
+.LINK
+https://github.com/TileDB-Inc/TileDB
+#>
+
+[CmdletBinding()]
+Param(
+)
+
+# Return the directory containing this script file.
+function Get-ScriptsDirectory {
+    Split-Path -Parent $PSCommandPath
+}
+
+$TileDBRootDirectory = Split-Path -Parent (Get-ScriptsDirectory)
+$InstallPrefix = Join-Path $TileDBRootDirectory "dist"
+$StagingDirectory = Join-Path (Get-ScriptsDirectory) "deps-staging"
+
+function DownloadFile([string] $URL, [string] $Dest) {
+    Write-Host "Downloading $URL to $Dest..."
+    Invoke-WebRequest -Uri $URL -OutFile $Dest
+}
+
+function DownloadIfNotExists([string] $Path, [string] $URL) {
+    if (!(Test-Path $Path)) {
+        # TLS1.2 seems to work for nodejs site
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+        DownloadFile $URL $Path
+    }
+}
+
+function run-batchfile-modifying-path([string] $scriptPath) {
+    Set-StrictMode -Version 3
+    $tempfile = [IO.Path]::GetTempFileName()
+    cmd /c " `"$scriptPath`" && set > `"$tempfile`" "
+    Get-Content $tempfile | ForEach-Object {
+        if($_ -match "^(PATH)=(.*)$") {
+            Set-Content "env:PATH" $matches[2]
+        }
+    }
+    Remove-Item $tempfile
+}
+
+
+function run-azurite() {
+    $azuriteDataPath= (Join-Path $env:TEMP "azurite-data")
+    $NodeJSGenericInstalledPath = Join-Path $InstallPrefix "NodeJS"
+    if(!(Test-Path $NodeJSGenericInstalledPath)) {
+	    Write-Host "Unable to find $NodeJSGenericInstalledPath!"
+	}
+    #presumes that the generic 'NodeJS' link to the versioned install directory exists.
+    $NodeJSInitBatchFile = (Join-Path $NodeJSGenericInstalledPath "nodevars.bat")
+    Write-Host "NodeJSGenericInstalledPath: " + $NodeJSGenericInstalledPath
+
+    #need this to have access to 'node' of nodejs...
+    run-batchfile-modifying-path $NodeJSInitBatchFile
+
+    #now actually run azurite
+    $npxCmdPath = (Join-Path $NodeJSGenericInstalledPath "npx.cmd")
+    $azuriteDebugLog = (Join-Path $azuriteDataPath "Debug.Log")
+    Write-Host "azuriteDebugLog: " + $azuriteDebugLog
+    #parameters for azurite-blob taken from those used in run-azurite.sh
+    #seems that npx(/npm?) is causing desired window title to be overwritten (title shown is 'npm')...?
+    #despite... https://stackoverflow.com/questions/30611803/how-to-set-the-title-of-a-windows-console-while-npm-is-running
+    $npxCmd = "start `"azurite host window`" " + $npxCmdPath + " azurite-blob" + " --silent --location " + $azuriteDataPath + " --debug " + $azuriteDebugLog + " --blobPort 10000 --blobHost 127.0.0.1 "
+    Write-Host "npxCmd: $npxCmd"
+    #this will leave a command prompt window open with nodejs/azurite running in it.
+    cmd /c $npxCmd
```

Comment:
Is there a reason to use `cmd` here rather than `Start-Process`?

Reply:
Because I could  not find any way with Start-Process to set a desired window title for the running process (azurite).
That said, the window title is currently not correct, because npm involved in the npx action apparently (re-)sets the window title (to 'npm') after it is invoked, but here's hoping in the future it might not do that.

Reply:
Marking resolved.